### PR TITLE
Added a small selection of wine to the recipes json

### PR DESCRIPTION
**Pull request by Jinko48**
Just added 3 different types of wine to the recipes json. This was more of a test than anything.

### CODE REVIEW

File: recipes/recipes_schubar.json [hunk 1]
```diff
@@ -841,6 +841,30 @@
...
         "prep": "build",
         "garnish": "Flag of lemon peel and raspberries"
+    },
+
+    "Cayuga White": {
+        "info": "Vintage 2013",
+        "tag": "core",
+        "unit": "oz",
+        "prep": "cool as desired",
+        "glass": "wine",
```
Comment (twschum) [inline recipes/recipes_schubar.json:851]: I'll add in "white wine" and "red wine" as options

File: recipes/recipes_schubar.json [hunk 2]
```diff
@@ -841,6 +841,30 @@
         "ice": "cubed",
         "prep": "build",
         "garnish": "Flag of lemon peel and raspberries"
+    },
+
+    "Cayuga White": {
```
Comment (twschum) [inline recipes/recipes_schubar.json:846]: Recipes here are generic, so this should really be a single entry of "wine".
Then in your ingredients file you'd make sure that the category listed is Wine, then Ingredient as white wine, then the specific bottle and type for Kind

File: recipes/recipes_schubar.json [hunk 3]
```diff
@@ -841,6 +841,30 @@
...
+        "info": "Vintage 2013",
+        "tag": "core",
+        "unit": "oz",
+        "prep": "cool as desired",
```
Comment (twschum) [inline recipes/recipes_schubar.json:850]: "prep" is kind of a special attribute that has to come from a restricted list, so just use "pour"

File: recipes/recipes_schubar.json [hunk 4]
```diff
@@ -841,6 +841,30 @@
         "ice": "cubed",
         "prep": "build",
         "garnish": "Flag of lemon peel and raspberries"
+    },
+
+    "Cayuga White": {
+        "info": "Vintage 2013",
```
Comment (twschum) [inline recipes/recipes_schubar.json:847]: You'll need to have an "ingredients" attribute, take a look a A Dram to see a similar type example. You can make a Red Wine and White Wine "recipe" and that should cover it all nicely. When a use looks at the drink, they'll see the list of what's available

**Review by twschum — CHANGES_REQUESTED** [review]

File: recipes/recipes_schubar.json (hunk 4 above)
Comment (Jinko48) [reply recipes/recipes_schubar.json:847]: Thanks for the tips!

**Pull request closed without merging**